iron-autogrow-textarea.html

```diff
@@ -82,6 +83,21 @@
       box-shadow: none;
     }
 
+    textarea::-webkit-input-placeholder {
```

Comment:
Paper-input actually does: 
```
      input:-moz-placeholder {
        color: var(--paper-input-container-color, --secondary-text-color);
      }
```

So doesn't create a special custom property just for it. I think that's better (and more consistent)